uv/Dockerfile

```diff
@@ -205,6 +204,5 @@ RUN mkdir -p ~/.cargo && printf "[net]\ngit-fetch-with-cli = true\n" >> ~/.cargo
 # Install uv by copying from the official image
 COPY --from=uv /uv /uvx /usr/local/bin/
 
-COPY --chown=dependabot:dependabot cargo $DEPENDABOT_HOME/cargo
-COPY --chown=dependabot:dependabot common $DEPENDABOT_HOME/common
+COPY --chown=dependabot:dependabot --parents cargo common $DEPENDABOT_HOME/
```

Comment:
these two could be combined, but I didn't want to do that in the scope of this PR... as there's a bit more that could actually be combined...